pages/entity/_type/_.vue

```diff
@@ -98,14 +99,34 @@
 
     computed: {
       attribution() {
+        if (this.editorialDepiction) return this.editorialAttribution;
         return (!this.entity || !this.entity.depiction) ? null : this.entity.depiction.source;
       },
       depiction() {
+        if (this.editorialDepiction) return this.editorialDepiction;
         return (!this.entity || !this.entity.depiction) ? null : entities.getWikimediaThumbnailUrl(this.entity.depiction.id);
       },
+      depictionLinkTitle() {
+        return this.editorialDepiction ? this.$t('goToRecord') : this.$t('entityDepictionCredit');
+      },
       description() {
         return this.editorialDescription ? this.editorialDescription : entities.getEntityDescription(this.entity);
       },
+      editorialAttribution() {
+        return this.page.primaryImageOfPage['en-GB'].fields.url['en-GB'];
```

Comment:
Worth putting the 'en-GB' string into Vues Data Object? ie. `EN-GB: 'en-GB'`

Reply:
They have now in fact been removed due to the reworking of the Contentful API requests to not include all locales in a single entry.